e2e/process-services/widgets/text_widget.e2e.ts

```diff
@@ -0,0 +1,122 @@
+import { LoginPage } from '../../pages/adf/loginPage';
+import { TasksPage } from '../../pages/adf/process_services/tasksPage';
+import { Widget } from '../../pages/adf/process_services/widgets/widget';
+import CONSTANTS = require('../../util/constants');
+import TestConfig = require('../../test.config');
+import resources = require('../../util/resources');
+
+describe('Text widget', () => {
+
+    let loginPage = new LoginPage();
+    let processUserModel;
+    let taskPage = new TasksPage();
+    let widget = new Widget();
+    let alfrescoJsApi;
+    let appsActions = new AppsActions();
+    let appModel;
+    let app = resources.Files.WIDGET_CHECK_APP.TEXT;
+    let deployedApp, process;
+
+    beforeAll(async (done) => {
+        let users = new UsersActions();
+
+        alfrescoJsApi = new AlfrescoApi({
+            provider: 'BPM',
+            hostBpm: TestConfig.adf.url
+        });
+
+        await alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+
+        processUserModel = await users.createTenantAndUser(alfrescoJsApi);
+
+        await alfrescoJsApi.login(processUserModel.email, processUserModel.password);
+        appModel = await appsActions.importPublishDeployApp(alfrescoJsApi, resources.Files.WIDGET_CHECK_APP.file_location);
+
+        let appDefinitions = await alfrescoJsApi.activiti.appsApi.getAppDefinitions();
+        deployedApp = appDefinitions.data.find((currentApp) => {
+            return currentApp.modelId === appModel.id;
+        });
+        process = await appsActions.startProcess(alfrescoJsApi, appModel, app.processName);
+        loginPage.loginToProcessServicesUsingUserModel(processUserModel);
+        done();
+    });
+
+    beforeEach(() => {
+        let urlToNavigateTo = `${TestConfig.adf.url}/activiti/apps/${deployedApp.id}/tasks/`;
+        browser.get(urlToNavigateTo);
+        taskPage.filtersPage().goToFilter(CONSTANTS.TASK_FILTERS.MY_TASKS);
+        taskPage.formFields().checkFormIsDisplayed();
+    });
+
+    afterAll(async (done) => {
+        await alfrescoJsApi.activiti.processApi.deleteProcessInstance(process.id);
+        await alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+        await alfrescoJsApi.activiti.adminTenantsApi.deleteTenant(processUserModel.tenantId);
+        done();
+    });
+
+    it('[C268157] General Properties', async () => {
+        let label = widget.textWidget().getFieldLabel(app.FIELD.simpleText);
+        expect(label).toBe('textSimple*');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+        let placeHolder = widget.textWidget().getFieldPlaceHolder(app.FIELD.simpleText);
+        expect(placeHolder).toBe('Type something...');
+        widget.textWidget().setValue(app.FIELD.simpleText, 'TEST');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeFalsy();
+    });
+
+    it('[C268170] Min-max length properties', async () => {
```

Comment:
Could you change the name to : Should be possible to set min-max length properties for Text widget ?